CMakeLists.txt

```diff
@@ -4,3 +4,4 @@
 # Author: Saravanan Sekar <saravanan@linumiz.com>
 
 add_subdirectory_ifdef(CONFIG_SOC_SERIES_M48X m48x)
+add_subdirectory_ifdef(CONFIG_SOC_SERIES_M46X m46x)
```

Comment:
Please add a new line at the end of file.

Reply:
@stephanosio , thanks of your advice.
It's fixed in commit [d57098a](https://github.com/zephyrproject-rtos/hal_nuvoton/pull/5/commits/d57098abaf62c532f33ea1d8cd3c6f144ba0a046).

Reply:
> @stephanosio , thanks of your advice.
> It's fixed in commit [d57098a](https://github.com/zephyrproject-rtos/hal_nuvoton/pull/5/commits/d57098abaf62c532f33ea1d8cd3c6f144ba0a046).

Thanks. Please squash that commit into `hal: NuMaker M46x: support external dts or *.h of module hal-M46x` since that change was introduced in this very PR.

Reply:
I see and squash this commit into `hal: NuMaker M46x: support external dts or *.h of module hal-M46x`